tests/integration/build_test.go

```diff
@@ -1,19 +1,49 @@
 //go:build integration
-// +build integration
 
 package integration
 
 import (
 	"encoding/json"
+	"fmt"
+	"path/filepath"
 	"testing"
+	"time"
 
+	"get.porter.sh/porter/pkg/build"
+	"get.porter.sh/porter/pkg/cnab"
 	"get.porter.sh/porter/pkg/yaml"
 	"get.porter.sh/porter/tests"
 	"get.porter.sh/porter/tests/tester"
 	"github.com/Masterminds/semver/v3"
+	"github.com/carolynvs/magex/shx"
 	"github.com/stretchr/testify/require"
 )
 
+func TestBuild(t *testing.T) {
+	test, err := tester.NewTest(t)
+	defer test.Close()
+	require.NoError(t, err, "test setup failed")
+
+	bunPath := filepath.Join(test.RepoRoot, "tests/testdata/mybuns/*")
+	require.NoError(t, shx.Copy(bunPath, test.TestDir, shx.CopyRecursive))
+	test.Chdir(test.TestDir)
+
+	// Use a unique version for appversion so that docker doesn't cache the result and not print the value used in the Dockerfile
+	appversion := fmt.Sprintf("app.versio=%d", time.Now().Unix())
```

Comment:
small typo: i guess it should be `app.version` instead of `app.versio`

Reply:
Thanks for catching that, it actually pointed out a problem with how I was doing the test, so I've cleaned it up a bit more.